resources/candlepin-rspec-tests.sh

```diff
@@ -3,15 +3,19 @@
 env | sort
 echo
 
-docker pull docker-registry.usersys.redhat.com/candlepin/candlepin-${CANDLEPIN_DATABASE} || true
-echo "Docker pull completed"
-
 # The docker container test script will know to copy out
-echo "Using workspace: $WORKSPACE"
-mkdir -p $WORKSPACE/artifacts/
+echo "Using workspace: ${WORKSPACE}"
+mkdir -p ${WORKSPACE}/artifacts/
 
 # make selinux happy via http://stackoverflow.com/a/24334000
-chcon -Rt svirt_sandbox_file_t $WORKSPACE//artifacts/
+chcon -Rt svirt_sandbox_file_t ${WORKSPACE}//artifacts/
 
 # Run the spec tests
-docker run -Pt --rm -v $WORKSPACE/artifacts/:/artifacts/ docker-registry.usersys.redhat.com/candlepin/candlepin-${CANDLEPIN_DATABASE} cp-test ${CP_TEST_ARGS} -c "${sha1}"
+TEST_DB=""
+case $CANDLEPIN_DATABASE in
+  mysql) TEST_DB="-m";;
+  postgresql) TEST_DB="-p";;
+  oracle) TEST_DB="-o";;
+esac
+
+./docker/test $TEST_DB -c "cp-test ${CP_TEST_ARGS} -c ${sha1}" -n "rspec-tests-${sha1}"
```

Comment:
Since this file is included in several types of tests, and sha1 won't be unique among multiple parallel rspec tests, I think we should use `-n $BUILD_TAG` instead. (tried successfully in Vagrant)